control/migrations/0031_auto_20190716_0942.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.9 on 2019-07-16 07:42
```

Comment:
Pour info : Il y a une option `makemigrations --name` qui permet de mettre un nom au migrations. Je le dis juste pour info, pas besoin de changer...

Reply:
ah cool good to know! Merci